src/main/java/com/monumental/services/EmailService.java

```diff
@@ -0,0 +1,171 @@
+package com.monumental.services;
+
+import com.monumental.config.AppConfig;
+import com.monumental.models.User;
+import com.monumental.models.VerificationToken;
+import com.monumental.models.suggestions.BulkCreateMonumentSuggestion;
+import com.monumental.models.suggestions.CreateMonumentSuggestion;
+import com.monumental.models.suggestions.UpdateMonumentSuggestion;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.MessageSource;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+import java.util.Locale;
+
+@Service
+public class EmailService {
```

Comment:
I decided to move all of the emailing logic to one spot instead of having to use `userService` to send emails.